lib/matplotlib/figure.py

```diff
@@ -466,20 +463,26 @@ def _set_dpi(self, dpi, forward=True):
 
     def get_tight_layout(self):
         """
-        Return whether the figure uses :meth:`tight_layout` when drawing.
+        Return whether and how `~.tight_layout` is called when drawing.
         """
         return self._tight
 
     def set_tight_layout(self, tight):
         """
-        Set whether :meth:`tight_layout` is used upon drawing.
-        If None, the rcParams['figure.autolayout'] value will be set.
+        Set whether and how `~.tight_layout` is called when drawing.
 
-        When providing a dict containing the keys `pad`, `w_pad`, `h_pad`
-        and `rect`, the default :meth:`tight_layout` paddings will be
-        overridden.
-
-        ACCEPTS: [True | False | dict | None ]
+        Parameters
+        ----------
+        tight : bool or dict with keys "pad", "w_pad", "h_pad", "rect" or None
+            If a bool, sets whether to call `~.tight_layout` upon drawing.
+            If ``None``, use the ``figure.autolayout`` rcparam instead.
+            If a dict, pass it as kwargs to `~.tight_layout`, overriding the
+            default paddings.
+
+            ..
+                ACCEPTS: [ bool
```

Comment:
Does this require line continuations for the tooling to pick it up correctly?  I remember there being discussion about `ACCEPTS` and why it is under a rst comment, but none of the details...

Reply:
It's a comment so that it doesn't appear in the HTML docs (https://929-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure.set_tight_layout) because it's totally redundant there.

Tooling resolves this correctly:
```
In [2]: plt.setp(matplotlib.figure.Figure)
  agg_filter: a filter function, which takes a (m, n, 3) float array and a dpi value, and returns a (m, n, 3) array 
  alpha: float (0.0 transparent through 1.0 opaque) 
  animated: bool 
  canvas: a FigureCanvas instance 
  clip_box: a `~.Bbox` instance 
  clip_on: bool 
  clip_path: [(`~matplotlib.path.Path`, `~.Transform`) | `~.Patch` | None] 
  contains: a callable function 
  dpi: float 
  edgecolor: any matplotlib color - see help(colors) 
  facecolor: any matplotlib color - see help(colors) 
  figheight: float 
  figure: a `~.Figure` instance 
  figwidth: float 
  frameon: boolean 
  gid: an id string 
  label: object 
  path_effects: `~.AbstractPathEffect` 
  picker: [None | bool | float | callable] 
  rasterized: bool or None 
  size_inches: a w, h tuple with w, h in inches
  sketch_params: (scale: float, length: float, randomness: float) 
  snap: bool or None 
  tight_layout: [ bool | dict with keys "pad", "w_pad", "h_pad", "rect" | None ] 
  transform: `~.Transform` 
  url: a url string 
  visible: bool 
  zorder: float
```

Reply:
^Hmmm, though 
```
transform: `~.Transform`
```
is mysterious enough that its an argument for being more verbose: `~matplotlib.transforms.Transform`

Reply:
I think the likelihood of someone wanting to fiddle with the transform using only the docstring as docs (remember that the HTML docs are hyperlinked) and not realizing that they are defined in the transforms module is quite low.